File: 0066-plus-one/solution.cpp
class Solution {
public:
    vector<int> plusOne(vector<int>& digits) {
        int x = digits.size()-1;
        
        while(x>=0){
            if(digits[x]+1==10){
                digits[x] = 0;
            }
            else{
                digits[x] += 1;
                return digits;
            }
            --x;
        }
        vector<int> decimals(digits.size()+1);
        decimals[0] = 1;
        return decimals;
    }
};

File: 0066-plus-one/solution.cs
public class Solution {
    public int[] PlusOne(int[] digits){
        string str = string.Join("", digits);
        BigInteger num = BigInteger.Parse(str);
        str = Convert.ToString(num + 1);
        int[] ans = new int[str.Length];
        
        for(int j=0; j<str.Length ; j++){
            int numInStr;
            
            if (int.TryParse(str[j].ToString(), out numInStr)){
                ans[j] = numInStr;
            }
        }
        return ans;
    }
}
